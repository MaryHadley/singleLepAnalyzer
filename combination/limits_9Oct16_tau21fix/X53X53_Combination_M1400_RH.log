000 (1295360.000.000) 10/09 22:41:04 Job submitted from host: <192.168.63.249:41533>
...
001 (1295360.000.000) 10/09 22:41:05 Job executing on host: <192.168.63.235:42190>
...
006 (1295360.000.000) 10/09 22:41:14 Image size of job updated: 188480
	86  -  MemoryUsage of job (MB)
	87972  -  ResidentSetSize of job (KB)
...
006 (1295360.000.000) 10/09 22:46:15 Image size of job updated: 282768
	141  -  MemoryUsage of job (MB)
	143800  -  ResidentSetSize of job (KB)
...
006 (1295360.000.000) 10/10 00:01:22 Image size of job updated: 282768
	141  -  MemoryUsage of job (MB)
	143884  -  ResidentSetSize of job (KB)
...
006 (1295360.000.000) 10/10 19:28:26 Image size of job updated: 297264
	154  -  MemoryUsage of job (MB)
	157480  -  ResidentSetSize of job (KB)
...
005 (1295360.000.000) 10/10 19:33:15 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 20:50:23, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 20:50:23, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1594  -  Run Bytes Sent By Job
	307  -  Run Bytes Received By Job
	1594  -  Total Bytes Sent By Job
	307  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :    31864        1   3824811
	   Memory (MB)          :      154     3072      3072
...
