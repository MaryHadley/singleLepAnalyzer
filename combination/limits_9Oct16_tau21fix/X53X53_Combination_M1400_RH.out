/var/lib/condor/execute/dir_1789
Total number of likelihood evaluations: 1100000101
Total number of likelihood evaluations: 11000101
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1400_RH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1400_RH/quant-toys:0-sig-77e702ef97.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1400_RH/quant-data-sig-3f93e9c69b.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1400_RH
