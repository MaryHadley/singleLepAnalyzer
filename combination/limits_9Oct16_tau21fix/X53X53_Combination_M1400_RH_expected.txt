# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.069554   0.035179    0.14437  0.048187    0.10194
